Makefile.generated_files

```diff
@@ -12,8 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
You'll hear no argument from me. I'll give this another go around and report back.